addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -171,5 +171,6 @@ options.registry.StepsConnector = options.Class.extend({
                 return `M ${0.05 * width} ${hHeight * 1.2} Q ${width / 2} ${hHeight * 1.8}, ${0.95 * width - 6} ${hHeight * 1.2}`;
             }
         }
+        return '';
```

Comment:
Wouldn't it be clearer to use `default` here ?
```javascript
switch (type) {
    ...
    default:
        return '';
}
```

Reply:
Both are fine, I would leave that to the dev :wink: Personally, I prefer as @Guillaume-gdi did. In general, I also prefer this:
```javascript
function hello() {
    if (condition) {
        return A;
    }
    return B;
}
```
over this
```javascript
function hello() {
    if (condition) {
        return A;
    } else {
        return B;
    }
}
```

I find it more readable and robust (future-proof to refactoring) to always have a `return` at the root-level of a function if it is meant to return something always. But it's really a matter of preference :+1: